### ISSUE TITLE
Inconvenience with load_pibmunic - PIB-Munic

### ISSUE THREAD

**Issue by lauratregadas** [datazoompuc/datazoom.amazonia#158]
# Data from 2019 onward is not available.

### Context

The dataset was pibmunic and the package had recently been downloaded from GitHub.

### Code

```r
data <- load_pibmunic(raw_data = FALSE,
                      geo_level = "state",
                      time_period = 2019)
```

### Goals and Expectations

README file on GitHub informs data is only available until 2018. However, it would be useful to have more recent data.

**Comment by gnjardim** [comment]
We already have data until 2020, I'll work on it in the next few days

**Comment by gnjardim** [comment]
Fixed on PR #171
